### DISCUSSION TITLE
How to configure css variables using styles api?

### DISCUSSION THREAD

**Discussion by jeroen-hoekstra** [mantinedev/mantine#4884] [Q&A]
Is it possible to configure css variables like the example below?

```
      <Button
        variant="outline"
        styles={{
          root: {
            '--button-color': 'red',
            color: 'var(--button-color)',
          },
        }}
      >
        Save
      </Button>
```

This doesn't seem to work as the text is still blue.

[See example](https://stackblitz.com/edit/react-uvs711)

**Comment by rtivital** [comment]
You can find examples of customizing components styles with Styles API on this page, it includes CSS variables, `styles` and `classNames` props – https://mantine.dev/styles/styles-api/. Follow examples in documentation, and you will be able to apply any styles to components.
